src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -318,7 +318,11 @@ private void findUnitsThatCantMove(final Map<Territory, ProPurchaseTerritory> pu
           if (ta != null) {
             production = ta.getProduction();
           }
-          if (territoriesToDefendWithOneUnit.contains(t) && unitValue <= (production + 3)) {
+
+          // Only defend territories that either already have units (avoid abandoning territories)
+          // or where unit value is less than production + 3 (avoid sacrificing expensive units to block)
+          if (territoriesToDefendWithOneUnit.contains(t)
```

Comment:
Yeah, its better to over defend currently to avoid blitz attacks though it will only use land units. It always chooses the cheapest unit with the least possible moves (further up in the method is sorting and land check). This is mostly a short term solution as eventually there should be more logic evaluating whether its worth 'sacrificing' a unit to block a territory and value of doing vs not doing it. This logic however seems to cover things well at least 95% of the time and going any further would take some significant changes.

I think the comment probably handles that particular part of the method well enough and the '3' is really just a magic number mostly based on the minimum unit cost (infantry) on most maps. I don't think giving it a label is gonna help much but I can add it if you feel strongly about it. Its really to avoid having really expensive, high move units from blocking territories far from the majority of a players forces. An example is on Napoleonic you don't really want Marshals moving 3 spaces to defend a random empty territory just to end up dying.